erts/preloaded/src/erts_internal.erl

```diff
@@ -484,6 +485,49 @@ perf_counter_unit() ->
 is_system_process(_Pid) ->
     erlang:nif_error(undefined).
 
+-spec sys_proc_outstanding_requests_limit() -> pos_integer().
+
+sys_proc_outstanding_requests_limit() ->
+    erlang:nif_error(undefined).
+
+set_code_and_literal_cleaner_prio(Prio) ->
+    Ref1 = make_ref(),
+    erts_code_purger ! {change_prio, self(), Ref1, Prio},
+    Ref2 = make_ref(),
+    LAC = find_lac(),
+    LAC ! {change_prio, self(), Ref2, Prio},
+    [{code_purger, receive {Ref1, OP1} -> OP1 end},
+     {literal_area_collector, receive {Ref2, OP2} -> OP2 end}].
+
+find_lac() ->
+    %% It is one of the first processes created...
+    case find_lac_quick(0) of
+        Pid when is_pid(Pid) ->
+            Pid;
+        undefined ->
```

Comment:
Maybe this should crash?

Reply:
Perhaps, this is probably a premature optimization. I expect to remove this in the future (see another of my answers; the order of answers became quite random).